src/java.base/share/classes/java/lang/classfile/components/CodeStackTracker.java

```diff
@@ -37,7 +37,7 @@
  * and calculating max stack size.
  * <p>
  * Sample use:
- * <p>
```

Comment:
This `<p>` intends to add a break. So I think you can just remove this new paragraph, like you've done below.